### DISCUSSION TITLE
Changing the trending page area.

### DISCUSSION THREAD

**Discussion by marccarran** [ViewTube/viewtube#2796] [Ideas]
I noticed that a lot of he trending content is German, so putting two and two together, I imagine this was intentional.
So, perhaps a great suggestion would be to have the option to change the trending page?

**Comment by moisout** [comment]
It's not intentional, it's because my server is in Germany :D. I don't know if I can request for a different location, I'll have to look into it.

**Reply by marccarran** [reply]
Ahh I see!
I thought  ViewTube was a Piped alternative made with the same code, so I was expecting it could be done.
But I KNOW it can be done, in the name of the highly logical German efficiency!
I know you can do it...
Do it for Ritter Sport.
Do it for Alex Wright.
Do it for Fritz Cola.

It's far better looking then Piped and Piped Material.
**Big, sexy, thumbnails.** 
It's what the web based YouTube alternatives have been missing. 
I mean yes, Piped has it's advantages too, but I can live with the restriction of hourly updates. 
I can also be reasonable and not expect you to have all the features of Piped this early in production, like any reasonable person.

All of this has made me think, what makes a good "alternative" service or app?
I think a lot of creators miss the best option, to replicate the look and feel of the original app.
Sometimes, there is a look to a certain app that just works. 

I think, once you replicate the look of the original app/service, you can then try to replicate the features, then once you add more features, you have a win/win situation. 

Well, it's about time I should say, thank you for your hard work so far, and I hope to see future updates on new features and eventually more people recognising and appreciating your hard work.

I am very sure that one day, you will enter the alternative service hall of fame.

Peace out, from myself, your Brother from England.

Moisout das wunderkind!
